config/200-clusterrole.yaml

```diff
@@ -3,7 +3,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: knative-contour-core
+  name: knative-serving-contour
```

Comment:
Although `knative-serving-contour` is fine but this is not `knative-net-contour` or `net-contour-core`?

Reply:
My thinking was 

1) This is consistent with other ingresses
2)  I think it's important to have `serving` in the cluster role name since it's contributing to a `serving` specific aggregated role via the label `serving.knative.dev/controller: "true"`